build.sbt

```diff
@@ -214,6 +214,7 @@ lazy val docs = http4sProject("docs")
   .settings(ghpages.settings)
   .settings(tutSettings)
   .settings(
+    libraryDependencies <+= scalazVersion {szv => argonautShapeless(szv) },
```

Comment:
We are now recommending circe as our default JSON library (#584).  I like argonaut, too, and am much happier to have this than nothing.  But we should port it to circe in the long term.